tests/integration/security/sds_ingress/quic/ingress_test.go

```diff
@@ -96,6 +101,11 @@ func TestMtlsGatewaysWithQUIC(t *testing.T) {
 		NewTest(t).
 		RequiresSingleCluster().
 		Run(func(t framework.TestContext) {
+			err := ingressutil.WaitForIngressQUICService(t, inst.Settings().SystemNamespace)
+			if err != nil {
```

Comment:
Do we want to be skipping this test if the error returned isn't QUIC mixed service not being supported?

Reply:
@jaellio 

You're right.
I've modified it to a flow, where an error is check for specific "Unsupported" error event.